react/features/rtcstats/middleware.ts

```diff
@@ -176,6 +176,18 @@ MiddlewareRegistry.register((store: IStore) => (next: Function) => (action: any)
         }
         break;
     }
+    case CAMERA_OFF_TIMESTAMP: {
+        if (canSendRtcstatsData(state) && faceLandmarks && faceLandmarks.enableRTCStats) {
```

Comment:
feels like the if statement is also used bellow, could you wrap everything in a canSendLandmarkRtcstatsData function?